src/Gnuplot.jl

```diff
@@ -1140,9 +1140,9 @@ function driver(_args...; is3d=false)
         end
     end
 
-    (doDump)  &&  (execall(gp))
+    # (doDump)  &&  (execall(gp))
 
-    return nothing
+    return gp
```

Comment:
When using the common REPL (without IJulia...) this would trigger dumping the content of a `Session` structure to the terminal.  We should avoid this, possibly by defining an empty `show()` method to cope with this case.

Moreover, the `Session` type (or better, their concrete subtypes) are supposed to be private structures of the package: we don't want the user to gain access to them in order to avoid accidental malfunctions.

A possible alternative, as suggested in this [post](https://discourse.julialang.org/t/ann-gnuplot-v1-1-0-a-julia-interface-to-gnuplot/37341/26?u=gcalderone), is to define a structure such as:
```julia
structure SessionID
  sid::Symbol
end
```
and change `return gp` into `return SessionID(gp.sid)`.

In this case:
- an empty `show` method needs to be defined for the `SessionID` structure;
- the `show` methods requiring png and svg outputs should be defined on the `SessionID` structure (rather than on `Session`);
- within the `show` method the actual `Session` object can be retrieved with `getsession(sid)`.

Reply:
Yea, this was just the most basic way to get it started. Do you prefer `SessionID` over something like `PlotHandle` or something more descriptive since this will be what users see and interact with?

Reply:
I would prefer `SessionID` since this is how I call it in the manual. Besides, users would see `Gnuplot.SessionID` which clearly identify the context, and if we implement the "empty" show method for `SessionID` when running in the REPL, the user would never knew this structure exists...

I avoided `PlotHandle` and similar names throughout the code since there can be multiple plot in a single session (`multiplot`) or even no plot at all in the case of dry sessions.

Another option is `SessionHandle` but, again, this would differ from what is written in the docs, and `Handle` is 4 characters longer than `ID`.

Finally, we may consider a name containing `broadcast`, or `connector`, since this is what we use it for.  I don't find these names particularly pleasant, but I have no other objection beside aesthetics.